Apply more formal tone to usage-guidelines.md (#33)
CodeMC/Documentation: docs/usage-guidelines.md

## Project Standards

Projects hosted on the CodeMC Continuous Integration (CI) system must comply with certain standards in order to remain on the platform.

### Project Quality Standards

The code of your project should exhibit a high standard of quality and effort. This includes, but is not limited to, adherence to established coding principles such as **Don't Repeat Yourself (DRY)**, **SOLID** principles, and similar best practices. Additionally, code should be well-documented, particularly in complex areas, to aid in clarity and understanding.

### Project Utility

While hosting a Continuous Integration (CI) setup for your library or project can be beneficial, it is essential that projects hosted on CodeMC serve a broader purpose. Specifically, projects that are solely designed to assist in the development of other projects, without any value to the wider community, should not be hosted on our CI system.

### Project Maintenance

CodeMC is not intended as a permanent repository for inactive or outdated projects. As such, projects hosted on the platform must demonstrate active maintenance through regular updates. Maintenance includes, but is not limited to:

> - Committing code with descriptive messages.

Reviewer: ```suggestion
Creating commits with descriptive messages.
```

`Committing` as a present-tense verb feels off.

- Keeping dependencies up to date.
- Managing the project’s issue and pull request trackers, which includes closing issues, merging pull requests, labeling them appropriately, and assigning responsible individuals.

> We acknowledge that there may be valid reasons for a project to become stale. In such instances, we request that you inform us of the situation, ensuring we remain informed.

Reviewer: ```suggestion
We acknowledge that there are valid reasons for a project to become stale. In such instances, we request that you inform us of the situation to ensure we remain informed.
```

### Prohibition of Illegal Activities or Content

CodeMC is committed to fostering a welcoming and inclusive programming community. Consequently, we do not tolerate projects that encourage, contain, or support hate speech, racism, sexism, bigotry, harassment, or any form of illegal content. Should you encounter such projects on our platform, you are required to report them promptly so they can be addressed accordingly.

### English Documentation Requirement

> Although CodeMC accepts projects in languages other than English, we require that projects provide English-language documentation. This is to ensure that users can easily understand the purpose and functionality of the project. English documentation should be readily accessible, whether in the form of the project's README or other relevant materials.

Reviewer: ```suggestion
Although CodeMC accepts projects in languages other than English, we require that projects provide English-language documentation. As CodeMC is a predominately English community, staff and users must be able to understand your project easily. English documentation should be readily accessible, whether in the project's README or other relevant materials.
```

Little more truthful.

## Punishments

CodeMC reserves the right to impose penalties if it is determined that these guidelines have been violated. The following actions may be taken at our discretion:

- Restriction of access to our Nexus and CI services.
- Removal of builds and Nexus uploads.
- Removal of your project(s).
- Termination of your account(s).
- Partial or complete suspension of access to our services.

These actions may be implemented individually or in combination, depending on the severity of the violation. The order and severity of penalties will be determined at CodeMC's discretion.

## Guideline Updates and Opting Out

These guidelines are subject to change at any time. In the event of an update, we will notify users via our [Discord Server](https://discord.gg/AGcFMu6), unless technical constraints prevent such notification. 

> If these guidelines are updated, users will have a period of 7 days to opt out of the updated terms. To opt out, users must cease using our services. Continued use of our services after the 7-day period constitutes acceptance of the updated guidelines.

Reviewer: ```suggestion
If these guidelines are updated, users will have a period of 7 days to opt out of the updated terms. To opt-out, users must cease using our services and notify staff immediately. Continued use of our services after 7 days constitutes acceptance of the updated guidelines.

The date of the last update can be found at the bottom of this document.
```

Include staff notification for ceasing use of services.
